pkg/auth/providers/common/usermanager.go

```diff
@@ -278,8 +278,19 @@ func (m *userManager) EnsureClusterToken(clusterName, tokenName, description, ki
 		return "", err
 	}
 	logrus.Infof("Creating token for user %v", userName)
```

Comment:
This was already present, but could we move it to log only if create is successful, otherwise log "Using existing token for user %v"? Would be easier to debug.

Reply:
@kinarashah it either creates the token or it fails, there is no using the existing token since it could be hashed.

Reply:
Ah. sorry I tend to forget we are now deleting existing tokens and no longer using them.